src/notebooksyncagent.cpp

```diff
@@ -309,7 +316,7 @@ void NotebookSyncAgent::sendLocalChanges()
         const QString &uid = deleted[i];
         QString href;
         Q_FOREACH(const Reader::CalendarResource &resource, mReceivedCalendarResources) {
-            if (!resource.incidence.isNull() && (uid == resource.incidence->uid())) {
+            if (!resource.incidences.isEmpty() && (uid == resource.incidences.first()->uid())) {
```

Comment:
Does this work if user locally deletes an exception to recurring event (i.e. an instance with recurrence id)? Appears this tries to remove the series from the server on such case.

Btw, hard tab on three comment lines above.